packages/a11y-base/src/field-aria-controller.d.ts

```diff
@@ -34,7 +34,14 @@ export class FieldAriaController {
    *
    * To unlink the previous slotted label element, pass `null` as `labelId`.
    */
-  setLabelId(labelId: string | null): void;
+  setLabelId(labelId: string | null, fromUser: boolean | null): void;
+
+  /**
+   * Defines the `aria-label` attribute of the target element.
+   *
+   * To remove the attribute, pass `null` as `label`.
+   */
+  setAriaLabel(label: string | null): void;
```

Comment:
nit: let's move this method before `setLabelId` to keep the alphabetical order where possible.

Reply:
Done.